export/class-export-settings.php

```diff
@@ -35,7 +35,7 @@ protected function get_application_manager()
 				require_once(WMP_PLUGIN_PATH.'frontend/class-application.php');
 			}
 
-            return new WMobilePack_Application();
+            return new WMobilePack_Application(plugin_dir_url(__FILE__));
```

Comment:
This should be cast to a variable that's assigned at the root level folder of the plugin